Doc/library/curses.rst

```diff
@@ -515,7 +515,11 @@ The module :mod:`curses` defines the following functions:
 
    Retrieves the value set by :func:`set_escdelay`.
 
+   Availability: if the ncurses library is used and on NetBSD.
+
    .. versionadded:: 3.9
+   .. availability:: not available with stock AIX
```

Comment:
Oh, I did not know about the `availability`. Then you can remove the manual note above.

And seems `availability` is usually added before `versionadded`. So please move it up. Also please add a period at the end.

Reply:
Missed that. Will push in a moment, but waiting for current tests to finish.

Reply:
p.s., re: Docs: I keep seeing this, seems it does not like the Document

Warning, treated as error:
[whatsnew/changelog:17] "`" found in "curses.get_tabsize and `func:`curses.set_tabsize"
Makefile:49: recipe for target 'build' failed

The Documentation is:
```
.. function:: get_tabsize()

   Retrieves the value set by :func:`set_tabsize`.

Is a special keyword needed to tell it to ignore this supposed mistake?
```